x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/Match.java

```diff
@@ -35,11 +37,12 @@
 /**
  * Full text function that performs a {@link QueryStringQuery} .
  */
-public class Match extends FullTextFunction implements Validatable {
+public class Match extends FullTextFunction implements Validatable, TwoOptionalArguments {
```

Comment:
It seems like we don't need to implement `TwoOptionalArguments` for now.

Reply:
Oh, good catch! Fixed in 597951b9150202f6724c76ff645ce95c9c65abe1